src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -140,51 +140,54 @@ function<void(EMIT_ARGS)> runtime::gpu::GPU_Emitter::get_emit_function(const Nod
 
     return it->second;
 }
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
```

Comment:
Is this actively needed or was this just convenient as you developed?
It's somewhat unusual to see a warning turned off for an entire file.

Reply:
Good catch, thanks.